### PR TITLE
Update blog with post detailing events of the outage on October 6

### PR DESCRIPTION
**Pull request by caseyyh**
## Checklist:

- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [x] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [x] I have manually confirmed that all new links work.
- [x] I added aliases (i.e., redirects) for all filename changes.
- [x] If making css changes, I rebuilt the bundle.

### CODE REVIEW

File: themes/default/content/blog/post-mortem-2023-10-06/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - post-mortem
+    - outage
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md
+# for details, and please remove these comments before submitting for review.
+---
+
+Last Friday, Pulumi suffered a 24 minute outage during which we failed to process 74.7% of received requests.
```
Comment (scottslowe) [inline themes/default/content/blog/post-mortem-2023-10-06/index.md:43]:
```suggestion
At Pulumi, we understand that Pulumi Cloud plays an important role in how our customers address their infrastructure management challenges. As a result, we strive for the highest levels of availability and performance in Pulumi Cloud. Unfortunately, on Friday, October 6, 2023, Pulumi Cloud suffered a 24 minute outage during which we failed to process 74.7% of received requests. In this post, we'd like to share our findings on the root cause of this outage, and the steps we are taking to ensure this sort of outage doesn't happen again.
```

Comment (scottslowe) [reply themes/default/content/blog/post-mortem-2023-10-06/index.md:43]: Just one potential way to rewrite the opening. Right now it feels very abrupt.

**Review by scottslowe — CHANGES_REQUESTED** [review]

File: themes/default/content/blog/post-mortem-2023-10-06/index.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+# At least one author is required. The values in this list correspond with the
+# `id` properties of the team member files at /data/team/team. Create a file for
+# yourself if you don't already have one.
+authors:
+    - casey-huang
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - post-mortem
```
Comment (joeduffy) [inline themes/default/content/blog/post-mortem-2023-10-06/index.md:35]: Total nitpick:

```suggestion
    - postmortem
```

File: themes/default/content/blog/post-mortem-2023-10-06/index.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    - outage
```
Comment (joeduffy) [inline themes/default/content/blog/post-mortem-2023-10-06/index.md:36]:
```suggestion
```

**Review by scottslowe — APPROVED** [review]

**Pull request merged**